src/libs/costmap_2d/package.xml

```diff
@@ -42,6 +42,7 @@
   <depend>xmlrpcpp</depend>
   <depend>pluginlib</depend>
   <depend>libpcl-all-dev</depend>
+ 	<depend>message_filters</depend>
```

Comment:
Spacing is off

Reply:
I'm not sure why my formatting keeps getting off on these files (package.xml & CMakeLists.txt), I have to keep fixing this. Does opening it in gedit somehow autofix the formatting?

Reply:
I wouldn't think so, but I'd recommend a 'real' editor as you main choice anyhow. Then you can configure it to do the right thing.